codespell_lib/data/dictionary.txt

```diff
@@ -24532,7 +24542,10 @@ sertificated->certificated
 sertificates->certificates
 sertification->certification
 servce->service
+servced->serviced
 servces->services
+servcing->servicing
+servecing->servicing
 serveice->service
```

Comment:
And perhaps this correction warrants the variants adding too? :)

Reply:
Not sure why this was marked as resolved? I was suggesting adding `serveiced->serviced` etc.

Reply:
Because I misunderstood you :-)
I'll add your suggestions

Reply:
7882538c1af6b53a545f5fa5e1176a9cb7098bb0